webapi/Models/Response/MigrationResult.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+namespace CopilotChat.WebApi.Models.Response;
+
+/// <summary>
+/// $$$
+/// </summary>
+public class MigrationResult
+{
+    /// <summary>
+    /// $$$
+    /// </summary>
+    public string Title { get; set; } = string.Empty;
+
+    /// <summary>
+    /// $$$
+    /// </summary>
+    public string Message { get; set; } = string.Empty;
+
+    /// <summary>
+    /// $$$
+    /// </summary>
+    public string? Note { get; set; }
```

Comment:
Do these ever get set?

Reply:
Not in this PR.  They will for the memory migration...otherwise the defaults are used.